docs/modules/ROOT/pages/tutorials/getting-started.adoc

```diff
@@ -1,483 +1,186 @@
-    },
-    \"tenantRef\":{}
-}}}"
-----
-+
-[TIP]
+[source,bash]
 --
-This patch is needed because of the new feature implemented in the Operator in https://github.com/projectsyn/lieutenant-operator/pull/110[PR #110 "Add cluster template to tenant"]. It will be added to the API in https://github.com/projectsyn/lieutenant-api/issues/89[Issue #89 "Expose Cluster Template Feature in Tenant Objects"].
+kubectl --context k3d-lieutenant get tenants -n lieutenant
 --
 
-. Retrieve the registered Tenants via API and directly on the cluster
-+
-[source,shell]
-----
-curl -H "$LIEUTENANT_AUTH" "http://${LIEUTENANT_URL}/tenants"
-kubectl -n lieutenant get tenant
-kubectl -n lieutenant get gitrepo
-----
+The command above returns something like the following:
 
-. Register a *Lieutenant Cluster* via the API
-+
-[source,shell]
-----
-CLUSTER_ID=$(curl -s -H "$LIEUTENANT_AUTH" -H "Content-Type: application/json" -X POST \
-  --data "{
-            \"tenant\": \"${TENANT_ID}\",
-            \"displayName\": \"My first Project Syn cluster\",
-            \"facts\": {
-              \"cloud\": \"local\",
-              \"distribution\": \"k3s\",
-              \"region\": \"local\"
-            },
-            \"gitRepo\": {
-              \"url\": \"ssh://git@${GITLAB_ENDPOINT}/${GITLAB_USERNAME}/cluster-gitops1.git\"
-          }}" \
-  "http://${LIEUTENANT_URL}/clusters" | jq -r ".id")
-echo $CLUSTER_ID
-echo https://${GITLAB_ENDPOINT}/${GITLAB_USERNAME}/cluster-gitops1
-----
-+
-[TIP]
+[source]
 --
-If everything went well, the Lieutenant Operator created a new git repository under https://${GITLAB_ENDPOINT}/${GITLAB_USERNAME}/cluster-gitops1 which will be used to store the generated catalog of deployment files.
+NAME             DISPLAY NAME         AGE
+t-aged-pond-24   Project Syn Tenant   13s
 --
 
-. Retrieve the registered Clusters via API and directly on the cluster
-+
-[source,shell]
-----
-curl -H "$LIEUTENANT_AUTH" "http://${LIEUTENANT_URL}/clusters"
-kubectl -n lieutenant get cluster
-kubectl -n lieutenant get gitrepo
-----
-
-== Kickstart Commodore
-
-Commodore is the configuration generation tool. It will be configured to generate configuration for your Lieutenant cluster `$CLUSTER_ID` generated above. With all the information available in Lieutenant, Commodore is able to figure out what to actually compile for the cluster in question and where to Git push the compiled catalog to.
+In the context of Project Syn, a "tenant" is an entity to whom clusters are assigned. Tenants could be customers, departments, teams, or any entity responsible for owning a cluster.
 
-Before continuing with this section, make sure that everything went well with the installation and configuration of Lieutenant as Commodore relies on having a working instance of it.
+Project Syn stores cluster-specific configuration values in that tenant's own configuration Git repository. You should now have a new project in your GitLab account called "Project Syn Tenant."
```

Comment:
Was there already an introduction to the hierarchy? I find it important to understand, that the configuration is a hierarchical aggregation of several sources.

Reply:
What difference does it make that the information is flat or hierarchical? At this point, we should only care that it exists. Again, this is a getting started guide, so less is more.

Reply:
Valid argument. I do not have a strong opinion here. It was just the first thoguht I had.

Reply:
Perfect then, resolving this item